tests/ui/if_not_else_bittest.rs

```diff
@@ -0,0 +1,11 @@
+#![deny(clippy::if_not_else)]
```

Comment:
Why is this in its own file? You can add it to `if_not_else.rs` instead.

Reply:
It is a separate test (absence of a warning), so putting it in its own file seems more appropriate. If you insist, I can mash it in the other file.

Reply:
No, that's alright.